exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillOperatorTable.java

```diff
@@ -43,24 +46,49 @@
 public class DrillOperatorTable extends SqlStdOperatorTable {
 //  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DrillOperatorTable.class);
   private static final SqlOperatorTable inner = SqlStdOperatorTable.instance();
-  private List<SqlOperator> operators = Lists.newArrayList();
+  private final List<SqlOperator> operatorsCalcite = Lists.newArrayList();
+  private final List<SqlOperator> operatorsDefault = Lists.newArrayList();
+  private final List<SqlOperator> operatorsInferernce = Lists.newArrayList();
   private final Map<SqlOperator, SqlOperator> calciteToWrapper = Maps.newIdentityHashMap();
-  private ArrayListMultimap<String, SqlOperator> opMap = ArrayListMultimap.create();
+
+  private final ArrayListMultimap<String, SqlOperator> opMapDefault = ArrayListMultimap.create();
+  private final ArrayListMultimap<String, SqlOperator> opMapInferernce = ArrayListMultimap.create();
+
+  private final SystemOptionManager systemOptionManager;
 
   public DrillOperatorTable(FunctionImplementationRegistry registry) {
+    this(registry, null);
```

Comment:
Why do we have this constructor, with systemOptionManager = null? Can we remove it ?